chainer/functions/loss/black_out.py

```diff
@@ -27,14 +27,33 @@ def black_out(x, t, W, samples):
        p(y) = \\frac{\\exp(W_y^\\top x)}{
        \\sum_{s \\in samples} \\exp(W_s^\\top x)}.
 
+    The output is a variable whose value depends on the value of
+    the option ``reduce``. If it is ``'samplewise'``, it holds the
+    samplewise loss values. If it is ``'mean'``, this function takes
+    a mean of loss values.
+
     Args:
         x (~chainer.Variable): Batch of input vectors.
+            Its shape should be :math:`(N, D)`.
```

Comment:
I forgot it but do we have a policy for which to use? I think it is better to align it with others.